### DISCUSSION TITLE
Newbie Questions.

### DISCUSSION THREAD

**Discussion by ritsp** [nathanmarlor/foxess_modbus#116] [Q&A] [answered]
Hello, Firstly what a great idea and work to integrate this. Sorry for some what maybe simple questions. 
I am setup and with the below sensors and switches. using the Foxcloud integration
![image](https://user-images.githubusercontent.com/24723476/230023651-9a2011c2-28a8-4514-bf5c-5691544dc68f.png)
 
My questions are. 
1. Will this now automatically "control" when my batteries charge to optimise the solar generation ? or do i need to enable something ?
2. i see an item saying the integration is "off" how do i enable this if i need to. 
![image](https://user-images.githubusercontent.com/24723476/230024619-58334daf-c043-4f3f-91da-c98cbbe4c6a4.png)
3. i have panels on 3 different facing directions... are we able to get those all in the single view ? or do i just use the 1 that generates most in the solcast integration

Sorry i these are answered somewhere.

**Comment by canton7** [comment, marked as answer]
I think you might be in the wrong repository? It looks like you're using foxess_em, which is here: https://github.com/nathanmarlor/foxess_em
